apps/service_profile/test/profile/feed/flow_test.exs

```diff
@@ -0,0 +1,109 @@
+defmodule Profile.Feed.FlowTest do
+  use ExUnit.Case
+  require Temp.Env
+
+  @instance Profile.Application.instance()
+
+  Temp.Env.modify([
+    %{
+      app: :service_profile,
+      key: Profile.Feed.Flow,
+      set: [
+        window_limit: 2,
+        window_unit: :second
+      ]
+    }
+  ])
+
+  setup do
+    Brook.Test.clear_view_state(@instance, "profile_stats")
+
+    :ok
+  end
+
+  test "aggregates message over a configured period and calls to configured reducers" do
+    dataset_id = "ds1"
+    subset_id = "sb1"
+
+    reducers = [
+      Profile.Reducer.MinMax.new(path: [Access.key(:value), "count"]),
+      Profile.Reducer.LongestString.new(path: [Access.key(:value), "name"])
```

Comment:
assume these haven't been implemented or at least pushed up yet?

Reply:
or do you get these for free with Flow?

Reply:
they are in test/support